Formula/fastd.rb

```diff
@@ -26,21 +26,35 @@ class Fastd < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "libuecc"
-  depends_on "libsodium"
   depends_on "bison" => :build # fastd requires bison >= 2.5
   depends_on "pkg-config" => :build
   depends_on "json-c"
+  depends_on "libuecc"
```

Comment:
👁

Reply:
I can put it back how it was if you really, really want, but since I had to remove `libsodium` from the deps list anyway it seemed a good moment to shuffle it into a happy place 🙈.